commands/load_balancers.go

```diff
@@ -63,6 +63,7 @@ With the load-balancer command, you can list, create, or delete load balancers,
 		fmt.Sprintf("The load balancer's size, e.g.: `lb-small`. Only one of %s and %s should be used", doctl.ArgSizeSlug, doctl.ArgSizeUnit))
 	AddIntFlag(cmdRecordCreate, doctl.ArgSizeUnit, "", 0,
 		fmt.Sprintf("The load balancer's size, e.g.: 1. Only one of %s and %s should be used", doctl.ArgSizeUnit, doctl.ArgSizeSlug))
+	AddStringFlag(cmdRecordCreate, doctl.ArgLoadBalancerType, "", "", "The type of load balancer")
```

Comment:
It would be great if the help string provided a hint of what the acceptable values are. 

If this feature will not be available to all customers for some time still, we may want to consider hiding it from the help output. You could do that with:

```
cmdRecordCreate.Flags().MarkHidden(doctl.ArgLoadBalancerType)
```

Reply:
Yep this feature will still remain behind a feature flipper for now: will update as suggested.